move noise into hgcroc conditions table

tomeichlersmith opened the pull request:
use ADC=2.0 as the average noise, this is a conservative default that aligns with noise found in CMS HGCAL tests

tomeichlersmith committed:
move noise into hgcroc conditions table

use ADC=2.0 as the average noise, this is a conservative default that
aligns with noise found in CMS HGCAL tests

cmantill reviewed (approved):
Looks good from my side

tomeichlersmith committed:
revert to previous low-noise settings

switching the technical implementaiton and the physical constants is too
much all at once, so I'm keeping the technical implementation but moving
the physical constant for the noise back to its 0.02PE value
